linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -47,9 +48,10 @@ object Infos {
         superClass: Option[String] = None,
         interfaces: List[String] = Nil,
         methods: List[MethodInfo] = Nil,
+        referencedFieldClasses: List[String] = Nil,
```

Comment:
Nitpick: I would order that before `methods`. We typically do fields before methods.